app/views/application/menus/_organization_listings_menu.html.erb

```diff
@@ -31,5 +31,14 @@
         <%= t "application.navbar.inquiry_public_link" %>
       <% end %>
     </li>
+
+    <li role="presentation">
+      <%= link_to "/transfers/multi/step/1" do %>
```

Comment:
can we use routes helpers here?

Reply:
route helpers are my cryptonite, cannot figure it out

```
                                      GET|POST /transfers/multi/step/:step(.:format)            multi_transfers#step
               multi_create_transfers POST     /transfers/multi/create(.:format)                multi_transfers#create
 ```

:thinking:

Reply:
jeje :smile_cat: in this case, I'd use `:as` option ([docs](https://guides.rubyonrails.org/routing.html#naming-routes)):

```ruby
match "multi/step/:step", to: "multi_transfers#step", via: [:get, :post], as: :multi_transfers_step
post "multi/create", to: "multi_transfers#create", as: :multi_transfers_create
```

Then you will be able to call these paths via `multi_transfers_step_path(step: 1)` and `multi_transfers_create`.

Reply:
thanks! :raised_hands: